.github/workflows/ci-sharktank.yml

```diff
@@ -22,6 +22,43 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  test_punet:
```

Comment:
Please order workflow jobs from simplest to most complicated. New jobs testing specific subsets of tests should go at the _bottom_ of the file, not the top. This affects both code structure and the display format on GitHub: 
![image](https://github.com/user-attachments/assets/a67e1995-c044-40c4-ab12-b52b64af45eb)